tests/test_websockets.py

```diff
@@ -522,3 +628,21 @@ async def app(scope: Scope, receive: Receive, send: Send):
     with pytest.raises(RuntimeError):
         with client.websocket_connect("/") as websocket:
             websocket.send({"type": "websocket.connect"})
+
+
+def test_send_disconnect_no_code(test_client_factory):
```

Comment:
We need an explanation about this test, or maybe turn it into a `xfail`, with the code, and write the reason on the `xfail` decorator.

Reply:
I expect you mean the "test_send_no_disconnect" test.  I'm adding a doc string.
The original test was written to verify that the app would supply a 1005 code.
I think that is the correct code to be put in place, it is an indication that no close code
was present on the wire.  But that can be a future PR.  As it is, the test just makes sure a close is
received, even without a result code and verifies whatever the current behaviour is.